share/completions/nvim.fish

```diff
@@ -1 +1,60 @@
-complete -c nvim -w vim
+# Options copied from vim.fish
+complete -c nvim -s c -r -d 'Execute Ex command after the first file has been read'
+complete -c nvim -s S -r -d 'Source file after the first file has been read'
+complete -c nvim -l cmd -r -d 'Execute Ex command before loading any vimrc'
+complete -c nvim -s i -r -d 'Set the shada file location'
+complete -c nvim -s o -r -d 'Open horizontally split windows for each file'
+complete -c nvim -s O -r -d 'Open vertically split windows for each file'
+complete -c nvim -s p -r -d 'Open tab pages for each file'
+complete -c nvim -s q -r -d 'Start in quickFix mode'
+complete -c nvim -s r -r -d 'Use swap files for recovery'
+complete -c nvim -s t -xa '(__fish_vim_tags)' -d 'Set the cursor to tag'
+complete -c nvim -s u -r -d 'Use alternative vimrc'
+complete -c nvim -s w -r -d 'Record all typed characters'
+complete -c nvim -s W -r -d 'Record all typed characters (overwrite file)'
+complete -c nvim -s A -d 'Start in Arabic mode'
+complete -c nvim -s b -d 'Start in binary mode'
+complete -c nvim -s d -d 'Start in diff mode'
+complete -c nvim -s D -d 'Debugging mode'
+complete -c nvim -s e -d 'Start in Ex mode, execute stdin as Ex commands'
+complete -c nvim -s E -d 'Start in Ex mode, read stdin as text into buffer 1'
+complete -c nvim -s h -d 'Print help message and exit'
+complete -c nvim -s H -d 'Start in Hebrew mode'
+complete -c nvim -s L -d 'List swap files'
+complete -c nvim -s m -d 'Disable file modification'
+complete -c nvim -s M -d 'Disable buffer modification'
+complete -c nvim -s n -d 'Don\'t use swap files'
+complete -c nvim -s R -d 'Read-only mode'
+complete -c nvim -s r -d 'List swap files'
+complete -c nvim -s V -d 'Start in verbose mode'
+complete -c nvim -s h -l help -d 'Print help message and exit'
+complete -c nvim -l noplugin -d 'Skip loading plugins'
+complete -c nvim -s v -l version -d 'Print version information and exit'
+
+# Support for those is planned, but not all of them are implemented yet (February 2023)
+# For the unimplemented, nvim prints a corresponding message
+# https://neovim.io/doc/user/remote.html
+complete -c nvim -l remote -d 'Edit files on NVim server'
+complete -c nvim -l remote-expr -d 'Evaluate expr on NVim server'
+complete -c nvim -l remote-send -d 'Send keys to NVim server'
+complete -c nvim -l remote-silent -d 'Edit files on NVim server'
+complete -c nvim -l remote-wait -d 'Edit files on NVim server'
+complete -c nvim -l remote-wait-silent -d 'Edit files on Vim server'
+complete -c nvim -l serverlist -d 'List all Vim servers that can be found'
+complete -c nvim -l servername -d 'Set server name'
+
+# Options not included in vim.fish, also available in vim
```

Comment:
> ... not included in vim.fish ...

After this PR, aren't these in vim.fish as well? So this comment is obsolete.

Reply:
Yes, that's right. I will remove them if everything else is okay.
I wasn't sure how to structure the `nvim.fish` file since it shares a lot of code with the `vim.fish` file, so I decided to add a few maybe unnecessary comments so it's more comprehensible for the reviewers where the respective entries came from. (Forgot to mention that in the original post.)